src/views/instances/details/actions-dropdown-menu.tsx

```diff
@@ -53,6 +56,7 @@ export const ActionsDropdownMenu = ({ deploymentId, instanceId, stackName, refet
             {open && (
                 <Popover onClickOutside={togglePopover} reference={anchor} placement="bottom-start">
                     <Menu>
+                        {stackName !== 'dhis2-db' && <ViewInstanceMenuItem groupName={groupName} name={name} stackName={stackName} />}
```

Comment:
It's clear how the condition evaluates, but the code doesn't provide any hint about **why** this happens.
In this case, I think it'd make sense to have a constant `VIEWABLE_INSTANCE_TYPES` (could be in `src/views/instances/details/constants.ts`)

Then we can change the condition from `stackName !== 'dhis2-db'` to `!VIEWABLE_INSTANCE_TYPES.includes(stackName)`, which makes this code expressive & declarative while keeping it explicit.

Reply:
Same applies to the condition in the other file